tests/alerts/test_geomodel_location.py

```diff
@@ -158,6 +111,7 @@ class TestAlertGeoModel(GeoModelTest):
                         'country': 'US',
                         'latitude': 37.773972,
                         'longitude': -122.431297,
+                        'observed': '2017-01-01T05:44:00+00:00',
```

Comment:
Now that we're comparing timestamps when comparing the `default_alert` against what we get from the geomodel alert, we'll want to use `toUTC` to put the time into whatever timezone we're using (which I believe is why the tests are currently failing).